modules/protocol_gateways/setup_validation.sh

```diff
@@ -80,7 +80,12 @@ fi
 
 cluster_size="${gateways_number}"
 
-current_mngmnt_ip=$(weka local resources | grep 'Management IPs' | awk '{print $NF}')
+current_mngmnt_ip=$(weka local ps -v --no-header -o managementIps)
```

Comment:
a fix require update in go-cloud-lib
moved this part to go-cloud-lib in https://github.com/weka/terraform-gcp-weka/pull/294